### DISCUSSION TITLE
Create Scenario-Based Documentation

### DISCUSSION THREAD

**Discussion by sympmarc** [microsoft-search/pnp-modern-search#3020] [Ideas]
**Is your feature request related to a problem? Please describe.**

The current docs are great, but take a functional approach. Scenario-driven documentation would make the PnP Modern Search Web Parts useful to a broader audience. Originally discussed in #720.

**Describe the solution you'd like**

A [first pass at scenarios](../blob/main/docs/scenarios/index.md) is available now.

**Additional context**

## Potential Future Scenarios

These suggestions from the community would make good additions.

* Rolling up Hub News (in yet another way!) - Jim Love @jimmywim
* Hooking up to the Suite Bar Search (combined with Set-PnPSearchSettings) - Jim Love @jimmywim
* Creating a simple People Directory - Jim Love @jimmywim
* How to use the search results as a "content query" to show a list of sites - Marijn Somers @marijnsomers
* How to properly set the select fields (when they don't show up)!  I end up exporting page to pnp template file, tweaking the select fields using text editor, saving template, then importing the page back in. - Brian McCullough @bpmccullough

**Comment by kasperbolarsen** [comment]
Hi @sympmarc , which of these are still relevant? The People directory should be OK, and I have just updated the documentation re the "Selected Properties"
@Marijnsomers , can you expand on your idea?

**Comment by sympmarc** [comment]
I forgot all about this discussion! Dang, I wish I'd saved the links for these scenarios.

I think we can consider this "solved", though @Marijnsomers scenario would still be nice.

I'd like to add a scenario about tags, especially given your fix, @kasperbolarsen. It's a bit confusing how to set things up, and a scenario that runs through it would be helpful, IMO.
